File: python3/PlusMinus.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the plusMinus function below.
def plusMinus(arr):
    results = {'positive': 0, 'negative': 0, 'zero': 0}
    for x in arr:
        if x > 0:
            results['positive'] += 1
        elif x < 0:
            results['negative'] += 1
        else:
            results['zero'] += 1
    print(round(results['positive']/len(arr), 6))
    print(round(results['negative']/len(arr), 6))
    print(round(results['zero']/len(arr), 6))

    



if __name__ == '__main__':
    n = int(input())

    arr = list(map(int, input().rstrip().split()))

    plusMinus(arr)

File: java8/PlusMinus.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

public class Solution {

    // Complete the plusMinus function below.
    static void plusMinus(int[] arr) {
        double positive=0, negative=0, zero=0;

        for(int x = 0; x < arr.length; x++){
            if (arr[x] > 0){
                positive++;
            }
            else if(arr[x] < 0){
                negative++;
            }
            else{
                zero++;
            }
        }

        System.out.printf("%.6f\n%.6f\n%.6f", positive/arr.length, negative/arr.length, zero/arr.length);


    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        int n = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        int[] arr = new int[n];

        String[] arrItems = scanner.nextLine().split(" ");
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        for (int i = 0; i < n; i++) {
            int arrItem = Integer.parseInt(arrItems[i]);
            arr[i] = arrItem;
        }

        plusMinus(arr);

        scanner.close();
    }
}
